src/software/jetson_nano/display/screens/screen.py

```diff
@@ -0,0 +1,126 @@
+from PIL import ImageFont
+import subprocess
+
+"""
+All new screens will inherit from this class. This class will handles editing variables and maintaining
+the current action the cursor is hovering.
+"""
+
+
+class Screen:
+    def __init__(self, lcd_display, status_codes, actions, action_map, draw_screen):
+        self.lcd_display = lcd_display
+        self.font_size = 12
+        self.font = ImageFont.truetype(
+            "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf", self.font_size
```

Comment:
let's copy this .ttf font file into a resources folder and link to that to avoid an absolute dependency

Reply:
done